modules/knock_down/functions/KnockDown/fn_KnockDownAction.sqf

```diff
@@ -0,0 +1,56 @@
+// AUTHOR: StatusRed (EM-Creations.co.uk)
+if !(hasinterface) exitwith {};
+params ["_validWeapons", "_reviveAction"];
+
+private _prevAnim = animationState player; // Get the player's current animation
+private _nearestUnits = nearestObjects [player, ["Man"], 3]; // Get all units within range
+private _useUnits = []; // Instanatiate an array for units which can be targetted
+
+private _animToUse = "acts_miller_knockout";
+
+// Check that they have a suitable weapon to knock down with
+if ((primaryWeapon player) in _validWeapons) then { // If the player has a valid weapon
+	if !(_nearestUnits isEqualTo []) then { // If there's at least one person nearby
+		{
+			_unit = _x;
+			//_isOut = _unit getVariable "knockedDown";
+			_isOut = false; // The above isn't working at the moment
+			if (!_isOut && {(side player) != (side _unit)}) then { // If the unit is not the player, not on the same side and not already hit
+				_useUnits = _useUnits + [_unit]; // Add it to the targetable units array
+			};
+		} foreach _nearestUnits; // For each of the nearby men
+
+		if !(_useUnits isEqualTo []) then { // If there's at least one targetable unit
+
+			[player, _animToUse] remoteExec ["switchMove", 0]; // Hitting animation
+
+			// This handler is NOT being called
+			player addEventHandler ["AnimDone", {
```

Comment:
This handler doesn't seem to be getting called, testing this no animation whatsoever is seen.

Reply:
is the switchmove working? it checks for `_useUnits`

Can you use `cursorobject` instead of a `nearestObjects` command?

Reply:
> is the switchmove working? it checks for `_useUnits`
> 
> Can you use `cursorobject` instead of a `nearestObjects` command?

The code should be getting run although for some reason no diag_log, hint or systemChat calls in that block seem to be output.. But the block is definitely being run because nearby units are knocked out as intended.

I've changed it from switchMove to playMove as I found switchMove won't trigger AnimDone events but it's still not working..